packages/core/setupTests.ts

```diff
@@ -0,0 +1,47 @@
+import { beforeAll, vi } from 'vitest';
+
+function updateLocation(partialLocation: Partial<Location>) {
```

Comment:
The tests expects a certain localhost domain and we frequently call `history.pushState` / `history.replaceState` and check for call to `location.assign`.
This was easier to do in jest, so this file is to make this easier in vitest.